pkg/cli-runtime/client_test.go

```diff
@@ -34,7 +36,7 @@ func TestNewClient(t *testing.T) {
 	scheme := runtime.NewScheme()
 	clitestingresource.AddToScheme(scheme)
 
-	c := NewClient("testdata/.kube/config", "", scheme)
+	c := NewClient(filepath.Join("testdata", ".kube", "config"), "", scheme)
```

Comment:
path for the testdata folder remains the same across all tests so declare the var globally and use the same in all tests